### ISSUE TITLE
Import Times are in UTC Instead of Server Time

### ISSUE THREAD

**Issue by kgrote** [barrelstrength/craft-sprout-import#76]
### Description
Entries imported using the documented date format are imported in UTC time instead of the server's timezone:

```
"@model": "barrelstrength\\sproutbase\\app\\import\\importers\\elements\\Entry",
    "attributes": {
      "sectionId": 1,
      "typeId": 1,
      "authorId": 1,
      "slug": "entry-slug",
      "postDate": "2015-02-27 16:43:52",
      "expiryDate": null,
      "enabled": true
    }
```

A quick fix would be to update the docs to show the full date formatting:

```
"@model": "barrelstrength\\sproutbase\\app\\import\\importers\\elements\\Entry",
    "attributes": {
      "sectionId": 1,
      "typeId": 1,
      "authorId": 1,
      "slug": "entry-slug",
      "postDate": {
	  "date": "2/27/2015",
          "timezone": "America/Los_Angeles",
          "time": "4:43 PM"
      },
      "expiryDate": null,
      "enabled": true
    }
```

Additionally the "Weed" panel in the control panel shows the Import Dates in UTC rather than the local server time.

**Comment by BenParizek** [comment]
Thanks for reporting. We'll get this resolved in the next release.

**Comment by pdaleramirez** [comment]
@BenParizek This is not a bug. The default behavior if you import a date will be added in UTC in the database but if you wish to add it on different timezone you can have array format with `timezone` value.

**Comment by pdaleramirez** [comment]
I have modified the Weed page to display the uploaded date and time based on the user's time setting in the control panel. This will be on the next release.

**Comment by BenParizek** [comment]
@pdaleramirez I'm still seeing a discrepancy here.

If I generate a seed Entry at 2019-02-15 10:00am ET (GMT-5) it creates an Element in the `elements` table and a record in the `sproutimport_seeds` table. Both of those rows are created at the same time. The `dateCreated` column for each of them is 2019-02-15 15:00:00.

However, when I look at both of those item in the UI, I see two different times:

In the UI, the new Entry Element shows a Date Created of **10:00 AM**.

However, the item listed on the Weed index page shows a Date Created of Feb 15, 2019 **03:00 PM** (5 hours later, which is the saved UTC time). The Weed tab should be displaying the same output to the user as the Entry Element, which would convert that UTC time and adjust it to the users timezone.

**Comment by pdaleramirez** [comment]
I believe this issue has been fixed. Closing this issue

**Comment by BenParizek** [comment]
@kgrote This has been addressed in Sprout Import v1.0.0-beta.22. The Weed page should now output dates in the same way that Craft does elsewhere in the CP.

I've also added a doc page discussing import date formats:
https://sprout.barrelstrengthdesign.com/docs/import/importing-dates.html

I'll consider expanding that to be in the default examples across all examples but it's a start for now.

Thanks for the feedback.
